docs/manual/en/introduction/Import-via-modules.html

```diff
@@ -62,11 +62,64 @@ <h2>Importing the module</h2>
 		...
 		</code>
 
-		<h2>Caveats</h2>
-
+		<h2>Importable Examples</h2>
 		<p>
-			Currently it's not possible to import the files within the "examples/js" directory in this way.
-			This is due to some of the files relying on global namespace pollution of THREE. For more information see <a href="https://github.com/mrdoob/three.js/issues/9562" target="_blank">Transform `examples/js` to support modules #9562</a>.
+			The core of three.js is focused on the most important components of a 3D engine. Many other components like loaders or controls are part of the
+			examples directory. three.js ensures to keep these files in sync with the core but users have to import them separately if they are required
+			for their project. However, most of these files are no modules which makes their usage in certain cases inconvenient. In order to address this issue,
+			three.js is going to provide more and more files as modules in the [link:https://github.com/mrdoob/three.js/tree/master/examples/jsm examples/jsm] directory.
+			If you install three.js via npm, you can import them like so:
+		</p>
+		<code>
+		import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
```

Comment:
I know it's mentioned in the previous section so maybe it's not worth noting again but this type of "bare module" syntax will only work with build processes like webpack or parcel.

Reply:
Um I think this should be already clear from the existing document. If we notice this section is unclear for users, we can still add an additional note.